dev/bots/test.dart

```diff
@@ -0,0 +1,153 @@
+      script: p.join('test_smoke_test', 'pass_test.dart'),
+      printOutput: false,
+  );
+  await _runFlutterTest(automatedTests,
+      script: p.join('test_smoke_test', 'crash1_test.dart'),
+      expectFailure: true,
+      printOutput: false,
+  );
+  await _runFlutterTest(automatedTests,
+      script: p.join('test_smoke_test', 'crash2_test.dart'),
+      expectFailure: true,
+      printOutput: false
+  );
+  await _runFlutterTest(automatedTests,
+      script: p.join('test_smoke_test', 'syntax_error_test.broken_dart'),
+      expectFailure: true,
+      printOutput: false,
+  );
+  await _runFlutterTest(automatedTests,
+      script: p.join('test_smoke_test', 'missing_import_test.broken_dart'),
+      expectFailure: true,
+      printOutput: false,
+  );
+  await _runCmd(flutter, <String>['drive', '--use-existing-app', '-t', p.join('test_driver', 'failure.dart')],
+      workingDirectory: p.join(flutterRoot, 'packages', 'flutter_driver'),
+      expectFailure: true,
+      printOutput: false,
+  );
+
+  List<String> coverageFlags = <String>[];
+  if (Platform.environment['TRAVIS'] != null && Platform.environment['TRAVIS_PULL_REQUEST'] == 'false')
+    coverageFlags.add('--coverage');
+
+  // Run tests.
+  await _runFlutterTest(p.join(flutterRoot, 'packages', 'flutter'),
+      options: coverageFlags,
+  );
+  await _runAllDartTests(p.join(flutterRoot, 'packages', 'flutter_driver'));
+  await _runFlutterTest(p.join(flutterRoot, 'packages', 'flutter_test'));
+  await _runAllDartTests(p.join(flutterRoot, 'packages', 'flutter_tools'),
+      environment: <String, String>{ 'FLUTTER_ROOT': flutterRoot },
+  );
+
+  await _runAllDartTests(p.join(flutterRoot, 'dev', 'devicelab'));
+  await _runFlutterTest(p.join(flutterRoot, 'dev', 'manual_tests'));
+  await _runFlutterTest(p.join(flutterRoot, 'examples', 'hello_world'));
+  await _runFlutterTest(p.join(flutterRoot, 'examples', 'layers'));
+  await _runFlutterTest(p.join(flutterRoot, 'examples', 'stocks'));
+  await _runFlutterTest(p.join(flutterRoot, 'examples', 'flutter_gallery'));
+
+  await _runCmd(dart, <String>[p.join(flutterRoot, 'dev', 'tools', 'mega_gallery.dart')],
+      workingDirectory: flutterRoot,
+  );
+  await _runFlutterAnalyze(p.join(flutterRoot, 'dev', 'benchmarks', 'mega_gallery'),
+      options: <String>['--watch', '--benchmark'],
+  );
+
+  print('\x1B[32mDONE: All tests successful.\x1B[0m');
+}
+
+Future<Null> _runCmd(String executable, List<String> arguments, {
+    String workingDirectory,
+    Map<String, String> environment,
+    bool expectFailure: false,
+    bool printOutput: true,
+}) async {
+  String cmd = '${p.relative(executable)} ${arguments.join(' ')}';
+  print('>>> RUNNING in \x1B[34m${p.relative(workingDirectory)}\x1B[0m: \x1B[33m$cmd\x1B[0m');
+
+  Process process = await Process.start(executable, arguments,
+      workingDirectory: workingDirectory,
+      environment: environment,
+  );
+
+  if (printOutput) {

```

Comment:
@goderbauer do you remember why we even support suppressing output?

Reply:
It looks like it's because the bash code whence this was forked redirected stdout to `/dev/null`.  That in turn was originated in https://github.com/flutter/flutter/pull/3977.

I'm gonna send a PR to remove it.  Swallowing the stdio obfuscated a serious Windows issue last night.

Reply:
Sounds good.

Reply:
Sent https://github.com/flutter/flutter/pull/16666